astropy/coordinates/tests/test_velocity_corrs.py

```diff
@@ -302,8 +304,8 @@ def test_invalid_argument_combos():
     with pytest.raises(ValueError):
         scwattrs.radial_velocity_correction(timel)
 
-
 @pytest.mark.remote_data
+@pytest.mark.filterwarnings('ignore')
```

Comment:
How about just passing in the right `obstime` to `SkyCoord`? Or alternatively explicitly check that the warning gets raised.

Reply:
Thanks for this suggestion @mhvk - it revealed a bug! When you add an ```obstime``` to the ```SkyCoord``` we call ```SkyCoord.apply_space_motion```, which raises and exception because there's no velocity present....

Reply:
Ah, good!